site/profile/templates/index.html.epp

```diff
@@ -17,15 +17,18 @@
               <img src="img/apple.png" height="128" width="128" />
             <% } -%>
             <h1>System Info </h1>
-            <p>FQDN :
-                <%= $fqdn %>
+            <p>fqdn/certname:
+                <%= $trusted[certname] %>
```

Comment:
@maju6406 I'm drawing fqdn from trusted certname to highlight concept of certificate auth and fact that our certnames are based on fqdn in this environment.